#!/usr/bin/python3
def raise_exception():
    print("hello" + 10)
